.circleci/src/workflows/commit.yml

```diff
@@ -1,2 +1,6 @@
 jobs:
   - check
+  - build_android
+  - deliver_android:
+      requires:
+        - build_android
```

Comment:
```suggestion
        - check
        - build_android
```
Or even make it required for `build_android`

Reply:
good idea. Changed it.